### DISCUSSION TITLE
New BirdWeather ID Request Form

### DISCUSSION THREAD

**Discussion by timsterc** [mcguirepr89/BirdNET-Pi#1203] [General]
Hi Everyone!

With an increasing number of BirdWeather ID requests coming through - we've decided to make it an automated process that you're able to do yourself.

You’ll first need to create an account - https://app.birdweather.com/login

Once that’s done - you can go to - https://app.birdweather.com/account/stations
Make sure that the Latitude and Longitude match what is in your BirdNET-Pi configuration.

NOTE - by using your BirdWeather ID - you are consenting to sharing your soundscapes and detections with BirdWeather.

Cheers!
-Tim
Founder - BirdWeather

**Comment by pwagterberg** [comment]
Hi,
I have create an account and accidentally create two stations with the same Latitude and Longitude.
How can i delete one of the two stations?

Greetings,

Patrick Agterberg
BirdIngelum
